docs/static/releasenotes.asciidoc

```diff
@@ -55,12 +56,64 @@ This section summarizes the changes in the following releases:
 * <<logstash-8-0-0-alpha1,Logstash 8.0.0-alpha1>>
 
 
+[[logstash-8-13-0]]
+=== Logstash 8.13.0 Release Notes
+
+* Package `elastic_integration` plugin. https://github.com/elastic/logstash/pull/15769[#15769]
+* Flow metrics added `worker_utilization` to pipeline-level https://github.com/elastic/logstash/pull/15912[#15912]
+* Unbundled `logstash-input-imap` plugin https://github.com/elastic/logstash/pull/15985[#15985]
+
+[[notable-8.13.0]]
+==== Notable issues fixed
+
+* Ensure that the order of JVM options from both files and environment variables is respected https://github.com/elastic/logstash/pull/15997[15997]
+
+==== Plugins
+
+*Elastic_integration Filter - 0.1.6*
+
+* Using this filter you can process Elastic Integrations in Logstash.
+** Once configured to point to an Elasticsearch cluster, this filter will detect which Elastic Integration processing (if any) should be executed for each event by auto-detecting the event’s data-stream.
+** It will then perform that processing inside Logstash without transmitting the event to Elasticsearch. Events that are successfully handled will be tagged so that any downstream Elasticsearch output in the Logstash pipeline will avoid running the Elastic Integration again in Elasticsearch.
```

Comment:
The plugin section should reflect the changelog entry for the version bump, not a general description of the plugin. 

*Elastic_integration Filter - 0.1.6
https://github.com/elastic/logstash-filter-elastic_integration/pull/127/files

Reply:
We need the info in release notes, but that's not the right way to handle it. 
I'll make some suggestions in the PR .